template/cordova/lib/run.js

```diff
@@ -39,7 +39,7 @@ module.exports.run = function (options) {
 
     // parse arg
     var args  = nopt({
-        'archs': String,
+        'archs': [String],
```

Comment:
This just aligns `--archs` interpretation with `build` module (see [build.js#L156](https://github.com/apache/cordova-windows/blob/43b101a6cbd8392d61e51dca01f25296ae84c89b/template/cordova/lib/build.js#L156)). Also the logic below ([L65](https://github.com/apache/cordova-windows/pull/190/files#diff-31dc29ac3df976968b00e031ea8cde82R65)) translates `archs: String -> archs: String[]` so this change effectively won't change existing behavior of 'run' command